FirebaseRemoteConfig/Tests/Unit/RCNConfigDBManagerTest.m

```diff
@@ -504,6 +504,45 @@ - (void)testWriteAndLoadExperiments {
   [self waitForExpectationsWithTimeout:_expectionTimeout handler:nil];
 }
 
+- (void)testWriteAndLoadActivatedExperiments {
+  XCTestExpectation *updateAndLoadExperimentExpectation =
+      [self expectationWithDescription:@"Update and load experiment in database successfully"];
+
+  NSError *error;
+  NSArray *payload2 = @[ @"ab", @"cd" ];
+  NSData *payloadData2 = [NSJSONSerialization dataWithJSONObject:payload2
+                                                         options:NSJSONWritingPrettyPrinted
+                                                           error:&error];
+  NSDictionary *payload3 =
+      @{@"experiment_ID" : @"35667", @"experiment_activate_name" : @"activate_game"};
```

Comment:
New to iOS code - Are the different `payload` variables here supposed to represent different experiments? I don't recognize the structure / what ab" "cd" and "experiment_activate_name" are.

Reply:
Yeah it's to represent different ExperimentDescriptions. This is how other payloads are written too. I don't think the actual JSON contents is too important in this test b/c it just makes sure it can read and write to the DB.